### PR TITLE
Please take a look. Thank you.

### PR DESCRIPTION
**Pull request by SpiralCode28**
I have put examples of some of what we went over during the review... I wanted to get this in by today how it is, per deadline. It is what I had due to the missed time all last week and weekend.

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -13,39 +13,94 @@
...
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" },
+    ];
+    expect().toEqual([
+      "Ford Prefect is a hitchhiker.",
+      "Zaphod Beeblebrox is president of the galaxy.",
+      "Arthur Dent is a radio employee.",
+    ]);
+  });
+});
```
Comment (sjproctor) [inline code-challenges.test.js:38]: The test looks great!

File: code-challenges.test.js [hunk 2]
```diff
@@ -13,39 +13,94 @@
...
+    })
+  }
+    
+//     '${obj.name\
+//     .split(" ")
+//     .map(word => word.charAt(0)
+//     .toUpperCase() + word
+//     .slice(1)
+//     .join(" ")} is ${obj.occupation}.')
+// }
```
Comment (sjproctor) [inline code-challenges.test.js]: Nice. These are great process notes.

File: code-challenges.test.js [hunk 3]
```diff
@@ -13,39 +13,94 @@
...
 // Expected output: [ 2, 1, -1 ]
+  it("", () => {
+    expect().toEqual;
+  });
+});
 
+const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false];
+// Expected output: [ 2, 0, -1, 0 ]
+const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true];
+// Expected output: [ 2, 1, -1 ]
```
Comment (sjproctor) [inline code-challenges.test.js]: Since you have the test variables inside your test you can delete them here.

File: code-challenges.test.js [hunk 4]
```diff
@@ -13,39 +13,94 @@
...
 // b) Create the function that makes the test pass.
+
+// psuedocode here
+const sumOfACube = (array) => {
+  let number = 0
+  for(let i = 0; i < array.length; i++) {
+    number += array[i]**3
+  }
+  return number
+}
```
Comment (sjproctor) [inline code-challenges.test.js]: Nice!

File: interview-questions.md [hunk 5]
```diff
@@ -7,44 +8,95 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
-  Your answer:
+Your answer:
 
-  Researched answer:
+-
 
+Researched answer:
 
+- The RESTful routes that always need to be passed params are `show`, `update`, `destroy`. The reason is because those methods would need to know what hashes the user wants to execute upon while `index` is everything and `create` is making something completely new.
```
Comment (sjproctor) [inline interview-questions.md:27]: Great!

File: interview-questions.md [hunk 6]
```diff
@@ -7,44 +8,95 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+
+   10. As a user, if I click on "trash," it deletes the list -- if I click on "cancel," the buttons disappear and I see the last list I had opened.
+
+   11. As a user, at the bottom of the page, there is a link with text saying "Home."
+
+   12. As a user, I can click on the "Home" link and it navigates me back to the Home webpage.
+
+   13. As a user, I can see a link on the homepage that says, "Create a new Do It list".
+
+   14. As a user, I can click on the "Create a New List" link and it takes me to a webpage where I see a header with the text, "Create a new Do It list".
```
Comment (sjproctor) [inline interview-questions.md:109]: Nice work!

**Review by sjproctor — CHANGES_REQUESTED** [review]
This looks great so far. Can you take one more pass at any blank problems. I know you're doing a lot of catch up. I think you are very close. I am just seeing the function on #2, adding the expect statements on #3, and the "your answers" on a couple interview questions.

**Comment by SpiralCode28** [comment]
I will take another pass. Thank you for reviewing.

**Comment by SpiralCode28** [comment]
Please review changes. Everything should be completed.

**Review by sjproctor — APPROVED** [review]
Awesome!! Great work, Hannah!

**Pull request merged**